src/components/DevDummyLogin/index.js

```diff
@@ -226,6 +243,13 @@ const DevDummyLogin = ({resetClickTime = () => {}}) => {
         </RBSheet>
         <RBSheet height={heightBs} ref={dummyLoginRbSheetRef}>
           <Text>Choose an account you wish to login</Text>
+          <TextInput
+            placeholder="Custom ID"
+            value={dummyInput}
+            onChangeText={(value) => setDummyInput(value)}
+            style={S.dummyInput}
+          />
+          <Button title={'Login with custom id'} onPress={() => dummyLogin(dummyInput)} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of a custom ID input field is a good feature for testing purposes. However, ensure that this feature is only available in development environments and not in production as it could pose a security risk.


<!-- This is an auto-generated comment by OSS CodeRabbit -->